blockchain/chain_makers.go

```diff
@@ -227,7 +228,7 @@ func makeHeader(chain consensus.ChainReader, parent *types.Block, state *state.S
 		time = new(big.Int).Add(parent.Time(), big.NewInt(10)) // block time is fixed at 10 seconds
 	}
 
-	return &types.Header{
+	header := &types.Header{
```

Comment:
By the way, there are no callcites of `makerHeader` function. May I ask if this function is called wherefrom?

Reply:
Did you search the call sites by `makeHeader`, not `makerHeader` ?. I also checked this function is used for test where making sub blockchain from DB.

Reply:
I'm sorry. `makeHeader` is the correct one. I couldn't find its usage in any package except for unit tests. Then, where the `CalcBaseFee` is called?

Reply:
`CalcBaseFee()` is called in `worker` and `txPool`.
The code will be added in next PR (updated in #1382)

Reply:
You can find the code in temporary branch(https://github.com/kjeom/klaytn/blob/dynamicGasPriceForTest/work/worker.go#L542) for Test.
It is for next PR.

Reply:
Thank you